Classes/Query/Modifier/Faceting.php

```diff
@@ -101,7 +101,6 @@ public function modifyQuery(Query $query)
      */
     protected function buildFacetingParameters()
     {
-        $facetParameters = array();
```

Comment:
It is not required to initialize the variable if it is filled in the if AND the else part below